src/logic/useOperators.ts

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from 'react'
+import { Operator } from '@entur/sdk/lib/mobility/types'
+
+import service from '../service'
+
+async function fetchOperators(): Promise<Operator[]> {
+    return service.mobility.getOperators()
+}
+
+export default function useOperators(ids?: string[]): Operator[] {
+    const [operators, setOperators] = useState<Operator[]>([])
+
+    useEffect(() => {
+        fetchOperators()
+            .then((data) => {
+                if (ids && ids.length > 0) {
+                    return setOperators(
+                        data.filter((operator) => ids.includes(operator.id)),
+                    )
+                }
+                setOperators(data)
+            })
+            // eslint-disable-next-line no-console
+            .catch((error) => console.error(error))
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
2: ignoring suggestion to include prop in dep-array

Reply:
For å unngå disabling her, så kan du gjere useEffecten uavhengig av `ids`:
```
useEffect(() => {
        fetchOperators()
            .then((data) => {
                setOperators(data)
            })
            // eslint-disable-next-line no-console
            .catch((error) => console.error(error))
}, [])

return operators.filter((operator) => !ids || ids.includes(operator.id)),
```